content/guides/autodiscovery.md

```diff
@@ -48,155 +86,169 @@ By default, the Datadog Agent includes Autodiscovery support for:
 
-    docker service create \
-      --name dd-agent \
-      --mode global \
-      --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
-      --mount type=bind,source=/proc/,target=/host/proc/,ro=true \
-      --mount type=bind,source=/sys/fs/cgroup/,target=/host/sys/fs/cgroup,ro=true \
-      -e API_KEY=<YOUR API KEY> \
-      -e SD_BACKEND=docker \
-      -e SD_CONFIG_BACKEND=etcd \
-      -e SD_BACKEND_HOST=127.0.0.1 \
-      -e SD_BACKEND_PORT=4001 \
-      datadog/docker-dd-agent:latest
+The following etcd commands create an Apache check template equivalent to that from the previous section:
+
+~~~
+etcdctl mkdir /datadog/check_configs/httpd
+etcdctl set /datadog/check_configs/httpd/check_names '["apache"]'
+etcdctl set /datadog/check_configs/httpd/init_configs '[{}]'
+etcdctl set /datadog/check_configs/httpd/instances '[{"apache_status_url": "http://%%host%%/server-status?auto"}]'
+~~~
 
-### Template structure in key-value stores
+Notice that each of the three values is a list. Autodiscovery assembles list items into check configurations based on shared list indexes. In this case, it composes the first (and only) check from `check_names[0]`, `init_configs[0]` and `instances[0]`.
 
-After your Datadog Agent service has been configured to use your Autodiscovery configuration backend, you will need to store your configuration templates in the structure:
+The following alternative etcd commands create the same Apache template and add an [HTTP check](https://github.com/DataDog/integrations-core/blob/master/http_check/conf.yaml.example) template:
 
-    /datadog/
-      check_configs/
-        docker_image_0/
-          - check_names: ["check_name_0"]
-          - init_configs: [{init_config}]
-          - instances: [{instance_config}]
-        docker_image_1/
-          - check_names: ["check_name_1a", "check_name_1b"]
-          - init_configs: [{init_config_1a}, {init_config_1b}]
-          - instances: [{instance_config_1a}, {instance_config_1b}]
-        ...
+~~~
+etcdctl set /datadog/check_configs/httpd/check_names '["apache", "http_check"]'
+etcdctl set /datadog/check_configs/httpd/init_configs '[{}, {}]'
+etcdctl set /datadog/check_configs/httpd/instances '[{"apache_status_url": "http://%%host%%/server-status?auto"},{"name": "My service", "url": "http://%%host%%", timeout: 1}]'
+~~~
 
-Note that in the structure above, you may have multiple checks for a single container. For example you may run a Java service that provides an HTTP API, using the HTTP check and the JMX integration at the same time. To declare that in templates, simply add elements to the `check_names`, `init_configs`, and `instances lists`. These elements will be matched together based on their index in their respective lists.
+Again, the list orders matter. The HTTP check will only work if all its elements have the same index (1) across the lists (they do).
```

Comment:
The HTTP check will only work if all its elements have the same index (1) across the lists (they do).
  I don't understand this